src/Products/DetailedProductEntity.php

```diff
@@ -4,9 +4,49 @@
 
 class DetailedProductEntity extends SimpleProductEntity
 {
-    private int $category_id;
-    private int $width;
-    private int $height;
-    private int $depth;
-    private int $related;
+    protected int $category_id;
+    protected int $width;
+    protected int $height;
+    protected int $depth;
+    protected int $related;
```

Comment:
These should just be private as we have no intention of inheriting from this class